The relationship between two comparable numbers is often expressed as a *growth rate*. For example, the United States federal government employed 2,766,000 people in 2002 and 2,814,000 people in 2012 \footnote{http://www.bls.gov/opub/mlr/2013/article/industry-employment-and-output-projections-to-2022-1.htm}. To compute a growth rate, we must first decide which value to treat as the `initial` amount. For values over time, the earlier value is a natural choice. Then, we divide the difference between the `changed` and `initial` amount by the `initial` amount.

In [10]:
initial = 2766000
changed = 2814000
(changed - initial) / initial

0.01735357917570499

A useful property of growth rates is that they don't change even if the values are expressed in different units. So, for example, we can express the same relationship between thousands of people in 2002 and 2012.

In [11]:
initial = 2766
changed = 2814
(changed - initial) / initial

0.01735357917570499

In 10 years, the number of employees of the US Federal Government has increased by only 1.74%. In that time, the total expenditures of the US Federal Government increased from \$2.37 trillion to \$3.38 trillion in 2012.

In [12]:
initial = 2.37
changed = 3.38
(changed - initial) / initial

0.4261603375527425

A 42.6% increase in the federal budget is much larger than the 1.74% increase in federal employees. In fact, the number of federal employees has grown much more slowly than the population of the United States, which increased 9.21% from 287.6 million people in 2002 to 314.1 million in 2012.

In [13]:
initial = 287.6
changed = 314.1
(changed - initial) / initial

0.09214186369958274

A growth rate can be negative, representing a decrease in some value. For example, the number of manufacturing jobs in the US decreased from 15.3 million in 2002 to 11.9 million in 2012, a -22.2% growth rate.

In [15]:
initial = 15.3
changed = 11.9
(changed - initial) / initial

-0.22222222222222224